runtime/amd64.S

```diff
@@ -732,6 +732,16 @@ FUNCTION(caml_c_call_stack_args)
 CFI_STARTPROC
         CFI_SIGNAL_FRAME
         ENTER_FUNCTION
+#ifdef WITH_THREAD_SANITIZER
+        TSAN_SAVE_CALLER_REGS
+    /* Also save %r12 and %r13 */
```

Comment:
Maybe this comment should say *why* we're saving these registers, rather than only stating *what* we're doing

Reply:
True. In addition, I realized that they didn’t need to be saved at all. See #14255.